apps/frontpage/app/docs/[...slug]/page.tsx

```diff
@@ -79,14 +87,6 @@ export const generateStaticParams = () => {
 };
 
 export default async function Page({ params: { slug } }: PageProps) {
-  // If the page is an index page, redirect to the parent page
```

Comment:
We probably only want to redirect if the page is a non-heading index page that is also empty.

I could not find a way to determine empty `page.content`, as it is a React Element.